connectionpool.go

```diff
@@ -478,6 +480,7 @@ func (pool *hostConnPool) logConnectErr(err error) {
 // transition back to a not-filling state.
 func (pool *hostConnPool) fillingStopped(err error) {
 	if err != nil {
+		log.Error().Msgf("pool filling stopped for %q: %v\n", pool.host.ConnectAddress(), err)
 		if gocqlDebug {
 			pool.logger.Printf("gocql: filling stopped %q: %v\n", pool.host.ConnectAddress(), err)
```

Comment:
The driver is already using standard logger. Let's note why we went ahead with zerolog